Content.Client/_DV/Curation/UI/Cwoink/CwoinkControl.xaml.cs

```diff
@@ -0,0 +1,282 @@
+/// </summary>
+[GenerateTypedNameReferences]
+public sealed partial class CwoinkControl : Control
+{
+    [Dependency] private readonly IClientAdminManager _adminManager = default!;
+    [Dependency] private readonly IClientConsoleHost _console = default!;
+    [Dependency] private readonly IUserInterfaceManager _ui = default!;
+    [Dependency] private readonly IConfigurationManager _cfg = default!;
+    public CuratorCHelpUIHandler CHelpHelper = default!;
+
+    private PlayerInfo? _currentPlayer;
+
+    public CwoinkControl()
+    {
+        RobustXamlLoader.Load(this);
+        IoCManager.InjectDependencies(this);
+
+        var newPlayerThreshold = 0;
+        _cfg.OnValueChanged(CCVars.NewPlayerThreshold, (val) => { newPlayerThreshold = val; }, true);
+
+        var uiController = _ui.GetUIController<CHelpUIController>();
+        if (uiController.UIHelper is not CuratorCHelpUIHandler helper)
+            return;
+
+        CHelpHelper = helper;
+
+        _adminManager.AdminStatusUpdated += UpdateButtons;
+        UpdateButtons();
+
+        CuratorOnly.OnToggled += args => PlaySound.Disabled = args.Pressed;
+
+        ChannelSelector.OnSelectionChanged += sel =>
+        {
+            _currentPlayer = sel;
+            SwitchToChannel(sel?.SessionId);
+            ChannelSelector.PlayerListContainer.DirtyList();
+        };
+
+        ChannelSelector.OverrideText += (info, text) =>
+        {
+            var sb = new StringBuilder();
+
+            if (info.Connected)
+                sb.Append(info.ActiveThisRound ? '⚫' : '◐');
+            else
+                sb.Append(info.ActiveThisRound ? '⭘' : '·');
+
+            sb.Append(' ');
+            if (CHelpHelper.TryGetChannel(info.SessionId, out var panel) && panel.Unread > 0)
+            {
+                if (panel.Unread < 11)
+                    sb.Append(new Rune('➀' + (panel.Unread-1)));
+                else
+                    sb.Append(new Rune(0x2639)); // ☹
+                sb.Append(' ');
+            }
+
+            // Mark antagonists with symbol
+            if (info.Antag && info.ActiveThisRound)
+                sb.Append(new Rune(0x1F5E1)); // 🗡
+
+            // Mark new players with symbol
+            if (IsNewPlayer(info))
+                sb.Append(new Rune(0x23F2)); // ⏲
+
+            sb.AppendFormat((string)"\"{0}\"", (object?)text);
+
+            return sb.ToString();
+        };
+
+        // <summary>
+        // Returns true if the player's overall playtime is under the set threshold
+        // </summary>
+        bool IsNewPlayer(PlayerInfo info)
+        {
+            // Don't show every disconnected player as new, don't show 0-minute players as new if threshold is
+            if (newPlayerThreshold <= 0 || info.OverallPlaytime is null && !info.Connected)
+                return false;
+
+            return (info.OverallPlaytime is null
+                    || info.OverallPlaytime < TimeSpan.FromMinutes(newPlayerThreshold));
```

Comment:
Parentheses can be removed
```suggestion
            return info.OverallPlaytime is null
                    || info.OverallPlaytime < TimeSpan.FromMinutes(newPlayerThreshold);
```